core/checks.py

```diff
@@ -971,7 +971,7 @@ def services(self):
                 # make it a service
                 # NOTE: the @ is left out from the service name as it implies
```

Comment:
Update the documentation here as well:
```suggestion
                # NOTE: the @ is include in the service name as it implies
```